addon/custom-map-with-default.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+class CustomMapWithDefault extends Ember.MapWithDefault {
+  constructor(options) {
```

Comment:
I think the default constructor could just be inherited, without need for shadowing in this subclass